### PR TITLE
user model and initial views

### PR DESCRIPTION
**Pull request by john-dw**
Please check user model creation. Thanks

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -11,6 +11,11 @@
   </head>
 
   <body>
+  <%= render "shared/navbar" %>
+  <p class="notice"><%= notice %></p>
+  <p class="alert"><%= alert %></p>
+
     <%= yield %>
+    <%= render "shared/flashes" %>
```
Comment (JM0033) [inline app/views/layouts/application.html.erb:19]: What does this refers to?

File: config/initializers/devise.rb [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+
+  # ==> Controller configuration
+  # Configure the parent class to the devise controllers.
+  # config.parent_controller = 'DeviseController'
+
+  # ==> Mailer Configuration
+  # Configure the e-mail address which will be shown in Devise::Mailer,
+  # note that it will be overwritten if you use your own mailer class
+  # with default "from" parameter.
+  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
```
Comment (JM0033) [inline config/initializers/devise.rb:27]: I think you need to change that line to update to our email address.

**Review by JM0033 — CHANGES_REQUESTED** [review]
I think there's one thing to change with the proper email address

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (john-dw) [reply app/views/layouts/application.html.erb:19]: This is the styling of devise notices of logins. I think.

File: config/initializers/devise.rb (hunk 2 above)
Comment (john-dw) [reply config/initializers/devise.rb:27]: Let's leave it like that for now. Thank you.

**Pull request merged**